config/06-ce-bdii.conf

```diff
@@ -0,0 +1,20 @@
+###############################################################################
+#
+# HTCondor-CE BDII/GLUE Publication configuration file.
+#
+###############################################################################
+
+# For multi-CE sites, only one CE publishes certain values.
+# Elect the leader here, either via zookeeper or fqdn.
+HTCONDORCE_BDII_ELECTION = LEADER
+HTCONDORCE_BDII_LEADER = $(FULL_HOSTNAME)
+
+# For Zookeeper
+#HTCONDORCE_BDII_ELECTION = ZOOKEEPER
+#HTCONDORCE_BDII_ZKHOSTS =
+
+# BDII Static Info and VOs
+HTCONDORCE_VONames = atlas, cms, alice, lhcb, dteam
+HTCONDORCE_SiteName = SITENAME
+HTCONDORCE_HEPSPEC_INFO = 10.00-HEP-SPEC06
+HTCONDORCE_CORES = 16
```

Comment:
Hi Iain. Just a quick question: `HTCONDORCE_CORES` refers to the total amount of cores in the cluster, is that correct?